src/drivers/vmount/output_mavlink.cpp

```diff
@@ -54,7 +54,7 @@ OutputMavlink::OutputMavlink(const OutputConfig &output_config)
 
 int OutputMavlink::update(const ControlData *control_data)
 {
-	vehicle_command_s vehicle_command;
+	vehicle_command_s vehicle_command = {};
```

Comment:
You can do it like this. 
https://github.com/PX4/Firmware/blob/master/src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp#L463